crates/sqlexec/src/remote/planner.rs

```diff
@@ -56,14 +57,18 @@ use crate::planner::physical_plan::show_var::ShowVarExec;
 use crate::planner::physical_plan::update::UpdateExec;
 
 use super::client::RemoteSessionClient;
-use super::rewriter::DDLRewriter;
 
 pub struct DDLExtensionPlanner {
+    catalog: SessionCatalog,
     catalog_version: u64,
```

Comment:
Is there a reason we have `catalog_version` stored here? Can't we simply `catalog.version()` where it's required.